admin_guide/topics/proc_adding-hosts.adoc

```diff
@@ -135,10 +135,10 @@ $ ansible-playbook [-i /path/to/file] \
     playbooks/openshift-master/scaleup.yml
 ----
 +
-. Set the node label to `logging-infra-fluentd: "true"`.
+. Set the node label to `logging-infra-fluentd=true`, if you installed a logging component on an existing cluster.
```

Comment:
We call logging the "EFK" stack elsewhere in the docs. Could you use that terminology?  I was confused at first by "a logging component."
Something like:
s/if you installed a logging component on an existing cluster/if you deployed the EFK stack in your cluster.

Reply:
I've modified as your point. Thanks !!